src/components/LandingLabAbout/index.tsx

```diff
@@ -43,13 +44,19 @@ export const LandingLabAbout: FC = () => {
           <div className='grid grid-cols-1 sm:grid-cols-3 lg:grid-cols-2 gap-4 md:gap-8'>
             <div className='sm:col-span-2 lg:col-span-1'>
               <AboutTitle>Wir bauen Prototypen</AboutTitle>
-              <p className='text-lg mb-8'>Für das Gemeinwohl</p>
+              <p className='text-lg mb-8'>Für die Stadtgesellschaft</p>
               <div className='prose max-w-none'>
                 <p className='max-w-none'>
                   In der Technologiestiftung Berlin arbeiten Expert*innen aus
                   den Bereichen Frontend- und Backendentwicklung, Data Science,
-                  Visualisierung, UI/UX zusammen, um aufzuzeigen, wie die
-                  Gemeinschaft von Technologie profitieren kann. Oder so.
+                  Datenvisualisierung, sowie UI/UX zusammen, um zu evaluieren
+                  und aufzuzeigen, wie unsere Stadtgesellschaft von Technologie
+                  profitieren kann.
+                </p>
+                <p>
+                  Das CityLAB Berlin ist Hauptakteur bei der Umsetzung der
+                  Prototypen. Neugierig geworden? Dann besuche einige unserer
```

Comment:
I would move the sentence "Das CityLAB Berlin ist Hauptakteur bei der Umsetzung der Prototypen" to the end of the paragraph above and leave "Neugierig geworden? Dann bes..." alone. Also we could add a link to "Projekte auf GitHub"